src/js/base/compat.js

```diff
@@ -71,8 +71,15 @@ if (
         const mixedModuleName = "Xmlhttprequest";
         const correctModuleName = mixedModuleName.toLowerCase();
         XMLHttpRequest = require(correctModuleName).XMLHttpRequest;
-    } else {
+    } else if (typeof window !== "undefined") {
         XMLHttpRequest = window.XMLHttpRequest;
+        // eslint-disable-next-line camelcase
+    } else if (typeof __webpack_require__ !== "undefined") {
+        // using a plain require call to load the module, since using
+        // the webpack call will result in module not found by nuxt.js
+        // applications

```

Comment:
Let's keep it nuxt-agnostic
```suggestion
        // use a runtime 'require' call that is
        // not parsed by webpack
```